extensions/blocks/slideshow/slideshow.php

```diff
@@ -25,6 +25,7 @@
 function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	$dependencies = array(
 		'lodash',
+		'wp-escape-html',
```

Comment:
Should match current master:

https://github.com/Automattic/jetpack/blob/201ec52be66261a364f0805a6e44d25269d84793/extensions/blocks/slideshow/slideshow.php#L28